src/libraries/Common/src/System/Security/Cryptography/Asn1/RSAPrivateKeyAsn.xml

```diff
@@ -2,7 +2,8 @@
 <asn:Sequence
   xmlns:asn="http://schemas.dot.net/asnxml/201808/"
   name="RSAPrivateKeyAsn"
-  namespace="System.Security.Cryptography.Asn1">
+  namespace="System.Security.Cryptography.Asn1"
+  rebind="false">
```

Comment:
The rebind attribute should be defined in https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Security/Cryptography/Asn1/asn.xsd so it shows up in IntelliSense.  Looks like you want to add it to TypeDefType, right after namespace.

Reply:
ok, added